atomate/vasp/workflows/base/hubbard_hund_linresp.py

```diff
@@ -0,0 +1,639 @@
+# coding: utf-8
+
+import os
+import itertools
+
+from atomate.vasp.fireworks.core import StaticFW
+from fireworks import Workflow, Firework
+from atomate.vasp.powerups import (
+    add_tags,
+    add_additional_fields_to_taskdocs,
+    add_wf_metadata,
+    add_common_powerups,
+)
+from atomate.vasp.workflows.base.core import get_wf
+
+from atomate.vasp.firetasks.run_calc import RunVaspCustodian
+from atomate.common.firetasks.glue_tasks import PassCalcLocs
+from atomate.vasp.firetasks.glue_tasks import CopyVaspOutputs
+from atomate.vasp.firetasks.parse_outputs import (
+    VaspToDb, HubbardHundLinRespToDb
+)
+from atomate.vasp.firetasks.write_inputs import WriteVaspFromIOSet
+
+from atomate.utils.utils import get_logger
+
+logger = get_logger(__name__)
+
+from atomate.vasp.config import VASP_CMD, DB_FILE, ADD_WF_METADATA
+
+from atomate.vasp.workflows.presets.scan import wf_scan_opt
+from uuid import uuid4
+from pymatgen.io.vasp.sets import MPRelaxSet, MPStaticSet
+from pymatgen.io.vasp.inputs import Poscar, Incar
+from pymatgen.core import Lattice, Structure, Element
+
+import numpy as np
+
+__author__ = "Guy Moore, Martin Siron"
+__maintainer__ = "Guy Moore"
+__email__ = "gmoore@lbl.gov"
+__status__ = "Production"
+__date__ = "February 2020"
+
+__hubbard_hund_linresp_wf_version__ = 0.0
+
+module_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)))
+
+
+def get_wf_hubbard_hund_linresp(
+        structure, uis,
```

Comment:
`uis` could be spelled out, should also be optional (default provided in body of `__init__`)

Reply:
I left `uis` as a required input due to the fact that `LMAXMIX` must be specified by the user. However, should I add some simple logic to set LMAXMIX=4 if a d-block, LMAXMIX=6 if a f-block, element is present in the input `Structure`?

Reply:
I've added a simple statement to `__init__` to set `LMAXMIX`, but please let me know if you have a better solution

Reply:
Setting automatically based on d/f seems ideal